tools/cldr-apps/src/main/webapp/js/esm/cldrVote.js

```diff
@@ -0,0 +1,429 @@
+/**
+ * cldrVote: encapsulate Survey Tool voting interface
+ */
+import * as cldrAjax from "./cldrAjax.js";
+import * as cldrDom from "./cldrDom.js";
+import * as cldrEvent from "./cldrEvent.js";
+import * as cldrInfo from "./cldrInfo.js";
+import * as cldrLoad from "./cldrLoad.js";
+import * as cldrRetry from "./cldrRetry.js";
+import * as cldrStatus from "./cldrStatus.js";
+import * as cldrSurvey from "./cldrSurvey.js";
+import * as cldrText from "./cldrText.js";
+
+/**
+ * Wire up the button to perform a submit
+ *
```

Comment:
This whole new file is essentially functions moved verbatim from cldrSurvey, for encapsulation, in preparation for implementing `pending()` and doing some refactoring for modernization